anclafx/provide.go

```diff
@@ -23,17 +21,6 @@ func Provide() fx.Option {
 			chrysom.ProvideDefaultListenerReader,
 			chrysom.ProvideListenerClient,
 		),
-		touchstone.Gauge(
-			prometheus.GaugeOpts{
-				Name: chrysom.WebhookListSizeGaugeName,
-				Help: chrysom.WebhookListSizeGaugeHelp,
-			}),
-		touchstone.CounterVec(
-			prometheus.CounterOpts{
-				Name: chrysom.PollsTotalCounterName,
-				Help: chrysom.PollsTotalCounterHelp,
-			},
-			chrysom.OutcomeLabel,
-		),
+		chrysom.ProvideMetrics(),
```

Comment:
Move the tochstone options to `chrysom.ProvideMetrics()`, better separation of concerns.